package.json

```diff
@@ -94,22 +94,22 @@
     "@oclif/plugin-plugins": "2.1.0",
     "@oclif/plugin-update": "3.0.0",
     "@oclif/plugin-version": "1.0.4",
-    "@salesforce/core": "^3.7.5",
-    "@salesforce/sf-plugins-core": "^1.7.1",
+    "@salesforce/core": "^3.10.0",
+    "@salesforce/sf-plugins-core": "^1.8.0",
     "@sf/config": "npm:@salesforce/plugin-config@2.3.1",
     "@sf/deploy-retrieve": "npm:@salesforce/plugin-deploy-retrieve@1.2.0",
-    "@sf/env": "npm:@salesforce/plugin-env@1.1.1",
+    "@sf/env": "npm:@salesforce/plugin-env@1.2.0",
```

Comment:
@RodEsp I don't think this should be bumped yet. My understanding is that we're waiting for the `create` commands to be done before shipping the `delete` commands

Reply:
I'm waiting on confirmation from @mshanemc to see if we can get https://github.com/salesforcecli/plugin-env/pull/252 merged before the release today.

Reply:
Shane said it's not going to get merged today so I reverted `plugin-env` to `1.1.1`